bin/ch/Helpers.cpp

```diff
@@ -211,3 +211,382 @@ void Helpers::LogError(__in __nullterminated const char16 *msg, ...)
     fflush(stdout);
     va_end(args);
 }
+
+
+void Helpers::TTReportLastIOErrorAsNeeded(BOOL ok, char* msg)
+{
+    if(!ok)
+    {
+#ifdef _WIN32
+        DWORD lastError = GetLastError();
+        LPTSTR pTemp = NULL;
+        FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_ARGUMENT_ARRAY, NULL, lastError, 0, (LPTSTR)&pTemp, 0, NULL);
+        fwprintf(stderr, _u(": %s"), pTemp);
+#endif
+        fprintf(stderr, "msg is: %s", msg);
```

Comment:
if the message is utf8 and a baseline depends on an output from this interface, the output won't match cross platform.